### PR TITLE
fix readme issues

### PR DESCRIPTION
**Pull request by dakshvar22**

### CODE REVIEW

**Review by amn41 — APPROVED** [review]
thank you Daksh

**Review by ArjaanBuijk — CHANGES_REQUESTED** [review]
The link to the langgraph's bot does not work.

This is the correct one I believe:
https://langchain-ai.github.io/langgraph/tutorials/customer-support/customer-support/

**Comment by hugobowne** [comment]
good catch @ArjaanBuijk ! fixed.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # CALM Reimplementation of LangGraph's customer service bot
 
-This is a reimplementation in CALM of langgraph's [customer support](https://langchain-ai.github.io/langgraph/tutorials/chatbots/customer-support/) example. 
-There's a [YouTube video](https://www.youtube.com/watch?v=b3XsvoFWp4c) that provides a walkthrough. 
+This is a reimplementation in of langgraph's [customer support](https://langchain-ai.github.io/langgraph/tutorials/customer-support/customer-support/) example in Rasa's [CALM](https://rasa.com/docs/rasa-pro/calm/) paradigm 
+There's a [YouTube video](https://www.youtube.com/watch?v=b3XsvoFWp4c) that provides a walkthrough of langgraph's implementation.
```
Comment (dakshvar22) [inline README.md:4]:
```suggestion
This is a reimplementation of langgraph's [customer support](https://langchain-ai.github.io/langgraph/tutorials/customer-support/customer-support/) example in Rasa's [CALM](https://rasa.com/docs/rasa-pro/calm/) paradigm 
There's a [YouTube video](https://www.youtube.com/watch?v=b3XsvoFWp4c) that provides a walkthrough of langgraph's implementation. 
```

**Review by ArjaanBuijk — APPROVED** [review]

**Pull request merged**